server/config/config.js

```diff
@@ -0,0 +1,17 @@
+const config = {
+    production:{
+        SECRET: process.env.SECRET,
+        DATABASE: process.env.MONGODB_URI
+
+    },
+    default:{
+        SECRET: 'SUPERSECRETPASSWORD1447',
+        DATABASE: 'mongodb://localhost:27017/booksReview'
```

Comment:
These to be stored within a secret file, writing them to a  `.env`  file will do.